lib/shared/permissions.js

```diff
@@ -198,6 +198,13 @@ exports.elevateCommand = (command, options) => {
   // for now, we should make sure we double check if the error messages
   // have changed every time we upgrade `sudo-prompt`.
 
+  }).catch((error) => {
+    return _.includes(_.trim(error.message), 'is not in the sudoers file');
+  }, () => {
+    throw errors.createUserError({
+      title: 'Your user doesn\'t have enough permissions to proceed',
+      description: 'This application requires sudo permissions to interact with removable drives'
```

Comment:
The `removable drives` bit is only true if the user hasn't enabled unsafe mode ;-)
I wonder if `This application requires sudo permissions to be able to write to drives` would be more generic / appropriate?